starters/express-apollo-prisma/README.md

```diff
@@ -0,0 +1,190 @@
+# express-apollo-prisma starter kit
+
+This starter kit features Express, Typescript API setup
+
+## Table of Contents
+
+- [express-apollo-prisma starter kit](#express-apollo-prisma-starter-kit)
+  - [Table of Contents](#table-of-contents)
+  - [Overview](#overview)
+    - [Tech Stack](#tech-stack)
+    - [Included Tooling](#included-tooling)
+  - [Installation](#installation)
+    - [CLI (Recommended)](#cli-recommended)
+    - [Manual](#manual)
+  - [Commands](#commands)
+    - [Database and Redis](#database-and-redis)
+    - [Seeding](#seeding)
+    - [Reset infrastructure](#reset-infrastructure)
+    - [Production build](#production-build)
+    - [CORS Cross-Origin Resource Sharing](#cors-cross-origin-resource-sharing)
+  - [Project Structure](#project-structure)
+    - [Folder structure](#folder-structure)
+    - [MVC Pattern](#mvc-pattern)
+      - [Example directory](#example-directory)
+  - [Technologies](#technologies)
+    - [Express](#express)
+    - [Apollo Server](#apollo-server)
+    - [Caching](#caching)
+    - [Testing](#testing)
+
+## Overview
+
+### Tech Stack
+
+- [Express v4](https://expressjs.com)
+- [Apollo Server v4](https://typeorm.io)
+- [Prisma v4](https://www.prisma.io/)
+- [MySQL](https://www.mysql.com/)
+- [Redis](https://redis.io/)
+-
+
+### Included Tooling
+
+- [Jest](https://jestjs.io/) - Test runner
+- [TypeScript](https://www.typescriptlang.org/) - Type checking
+- [ESLint](https://eslint.org/) - Code linting
+- [Prettier](https://prettier.io/) - Code formatting
+
+## Installation
+
+### CLI (Recommended)
+
+```bash
+npm create @this-dot/starter --kit express-apollo-prisma
+```
+
+or
+
+```bash
+yarn create @this-dot/starter --kit express-apollo-prisma
+```
+
+- Follow the prompts to select the `express-apollo-prisma` starter kit and name your new project.
+- `cd` into your project directory and run `npm install`.
+- Make sure you have docker & docker-compose installed on your machine
+- Create a `.env` file and copy the contents of `.env.example` into it.
+- Run `npm run docker:mount` to start the database and the Redis instances
+- Run `npm run start` to start the development server.
+- Open your browser to `http://localhost:4001` to see the API documentation with the existing endpoints.
+
+### Manual
+
+```bash
+git clone https://github.com/thisdot/starter.dev.git
+```
+
+- Copy and rename the `starters/express-apollo-prisma` directory to the name of your new project.
+- Make sure you have docker & docker-compose installed on your machine
+- `cd` into your project directory and run `npm install`.
+- Make sure you have docker & docker-compose installed on your machine
+- Create a `.env` file and copy the contents of `.env.example` into it.
+- Run `npm run docker:mount` to start the database and the Redis instances
```

Comment:
Not introduced in this PR, but the first time I notice this command and might be a good PR to get it changed.

The script name `docker:mount` might be confusing to the users (it is to me) because it does more than just mounting. It starts the whole infrastructure required to run our stack locally. In the [express-typeorm-postgres package.json](https://github.com/thisdot/starter.dev/blob/main/starters/express-typeorm-postgres/package.json) we use `infrastructure:start` and `infrastructure:stop` instead.

Shall we take this opportunity to rename it in this kit for it to make sense and be consistent?

Reply:
Yes I guess I can take the opportunity to change `docker:mount` and `docker:unmount`. to `infrastcture:start` and `infrastructure:stop`